plugins/org.jboss.tools.cdi.reddeer/src/org/jboss/tools/cdi/reddeer/validators/AbstractValidationProvider.java

```diff
@@ -19,8 +19,14 @@
 public abstract class AbstractValidationProvider implements IValidationProvider {
 
 	protected List<ValidationProblem> problems = null;
+	protected String jsr;
```

Comment:
Why are you using this property in parent abstract class? In previous cases you have put it one class down, right?

Reply:
It has a reason - the init() function is called in the parent abstract class, when the jsr is initialized in the class, that extends the parent class. When the init() is called, the jsr would be uninitialized (results in NPE), if you let the jsr be set in constructor of the subclass.

Reply:
We agreed that it did not mind (was not testing JSR string in previous PR, so it passes tests) in merged PR. Continue with your approach and we will revise tests that are not taking in account JSR versions.